WIP | PR | The docker image has been created and the docker file has been attached

iScous opened the pull request:
Work In Progress: Follow documentation in README.md;
Issue: #9

arsenykonohovwork commented on README.md:
```diff
@@ -1,2 +1,17 @@
+  * Created a docker file with the correct instructions. It includes a package.json file with all the dependencies needed to install correctly node.js, npm and frameworks
+  * Created a docker image from the docker file
+  * Builded the docker image
+  * Pushed the docker image to the docker hub
```
@iScous You have to add examples of commands that we have to use for creating, deploy, run container;

iScous replied:
@arsenykonohovwork added the commands to the file.

arsenykonohovwork commented on Dockerfile:
```diff
@@ -0,0 +1,10 @@
+COPY ./index.js /src/index.js
+RUN npm install
+COPY . /app
+CMD node /src/index.js
```
@iScous You have to fix STDOUT output, cause we run process in container and can not walk through it;

iScous replied:
@arsenykonohovwork  so the src folder cannot be used?

arsenykonohovwork commented on Dockerfile:
```diff
@@ -0,0 +1,10 @@
+FROM node:8.12.0
+RUN mkdir /src
+WORKDIR /src
+COPY ./package.json /src/package.json
```
@iScous where is package.json in source code?

iScous replied:
@arsenykonohovwork you mean it must be attached?

iScous commented:
@arsenykonohovwork ready for review. Added package.json, enabled to navigate in the running container. And for this PR as README.md was added previously i remained it here and modified

iScous commented:
@arsenykonohovwork the docker-compose.yml was added

iScous commented on README.md:
````diff
@@ -1,2 +1,43 @@
+To specify multiple containers we can use docker-compose.yml file. To create and start containers it's possible to use the command:
+```sh
+docker-compose up -d
+```
````
I think yes. I'll try to do that

iScous commented:
@arsenykonohovwork made changes with relation to your latest comments

arsenykonohovwork commented:
@iScous this PR is ready for merge;

arsenykonohovwork commented:
But work still in progress;
So you (@iScous) shouldn't close related issue;
